### DISCUSSION TITLE
Add more themes or a section to customize my workspace.

### DISCUSSION THREAD

**Discussion by MrUprizing** [zed-industries/zed#13049] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the feature

Add more themes or a section to customize my workspace.

**Comment by maxdeviant** [comment]
There are lots of themes available as extensions, and you can also make or customize your own theme.

**Comment by Moshyfawn** [comment]
Be sure to browse through #7337 for inspiration/beautiful themes and consider reading up on [how to create your own theme](https://github.com/zed-industries/extensions/blob/main/AUTHORING_EXTENSIONS.md#themes-1).
